module/userProjects/helper.js

```diff
@@ -4076,10 +4079,31 @@ function _updateUserProfileBasedOnUserRoleInfo(userProfile, userRoleInformation)
                 userProfile.userRoleMismatchFoundAndUpdated = true;
             }
 
+
             // Create location only object from userRoleInformation
             let userRoleInformationLocationObject = _.omit(userRoleInformation, ['role']);
-            
+
+//check whether userRoleInformation(reqbody) and (db )data matches
+
+for (const key in userRoleInformationLocationObject) {
```

Comment:
if one data is missing in that case also it will pass @praveenKDass 
we need to validate on if any data is missing. in that case also.